### ISSUE TITLE
Warning after upgrade the hexo

### ISSUE THREAD

**Issue by newdee** [hexojs/hexo#2845]
after update the hexo，it always shows:
```
zone.offset has been deprecated in favor of zone.utcOffset
```
any help is welcome.

**Comment by JLHwung** [comment]
Please run `npm ls moment` to see which hexo plugin is using `moment`.

**Comment by khs1994** [comment]
```
⋊> ~/k/a/www.khs1994 on hexo ⨯ npm ls moment                                                                                                           22:37:49
www.khs1994.com@0.0.1 /Users/khs1994/khs1994-lnmp/app/www.khs1994
├─┬ hexo@3.4.1
│ ├─┬ hexo-log@0.2.0
│ │ └─┬ hexo-bunyan@1.0.0
│ │   └── moment@2.13.0  deduped
│ ├── moment@2.13.0
│ └─┬ moment-timezone@0.5.14
│   └── moment@2.13.0  deduped
└─┬ hexo-deployer-git@0.3.1
  └── moment@2.19.1
```

**Comment by newdee** [comment]
the result is:
```
hexo-site@0.0.0 /home/newdee/Blog
├─┬ hexo-deployer-git@0.3.1
│ └── moment@2.18.1 
├─┬ hexo-generator-baidu-sitemap@0.1.2
│ └─┬ hexo@3.4.0
│   ├─┬ hexo-log@0.2.0
│   │ └─┬ hexo-bunyan@1.0.0
│   │   └── moment@2.19.2  deduped
│   └── moment@2.13.0 
├── moment@2.19.2 
└─┬ moment-timezone@0.5.14
  └── moment@2.19.2  deduped

```

**Comment by innks** [comment]
hexo-site@0.0.0 D:\cloud-dev\innksBlogData
+-- hexo@3.4.1
| +-- hexo-log@0.2.0
| | `-- hexo-bunyan@1.0.0
| |   `-- moment@2.13.0  deduped
| +-- moment@2.13.0
| `-- moment-timezone@0.5.14
|   `-- moment@2.13.0  deduped
+-- hexo-deployer-git@0.3.1
| `-- moment@2.19.2
`-- hexo-generator-json-content@3.0.1
  `-- moment@2.19.2

Me too. how?

zone.offset has been deprecated in favor of zone.utcOffset

**Comment by guo-long** [comment]
Always show many lines: zone.offset has been deprecated in favor of zone.utcOffset
How to fix this problem?

**Comment by newdee** [comment]
@geekrainy  I think you can add the `bug` or `warning problem` tags, thank you.

**Comment by geekrainy** [comment]
I got the same error.

**Comment by JLHwung** [comment]
Could you run
```js
npm install moment@^2.19.2 moment-timezone@^0.5.14
```
to see if the error can be got rid of?

**Comment by newdee** [comment]
@JLHwung  it still shows the same problem

**Comment by JLHwung** [comment]
@newdee Will fix on next 3.4.2 release.
